pkg/ingester/transfer.go

```diff
@@ -695,15 +661,18 @@ func transferUser(ctx context.Context, stream client.Ingester_TransferTSDBClient
 				return err
 			}
 
-			if err := batchSend(1024*1024, b, stream, &client.TimeSeriesFile{
+			bytesSent, err := batchSend(1024*1024, b, stream, &client.TimeSeriesFile{
```

Comment:
Not sure if it is relevant, but if a single file transfer takes a long time, then the metric increment wont be reflected till we have sent the entire file. Do you think we should pass the metric in the function instead so that the gradual increment is visible?

Reply:
@gouthamve @pstibrany Opinion on this? I see a value in what Ganesh is mentioning, while returning the sent bytes looks cleaner on the code side.

Reply:
No strong opinions from my side, but at previous jobs our admins preferred to see ongoing transfers in metrics, not just big "jump" at the end of transfer. If it requires some non-trivial code changes, I'd suggest doing it in separate PR.

Reply:
Right. Let's get stick to the previous behaviour. I restored it, tracking the metric directly in `batchSend()`.